sbnanaobj/StandardRecord/SRCRUMBSResult.h

```diff
@@ -0,0 +1,42 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRCRUMBSResult.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRCRUMBSRESULT_H
+#define SRCRUMBSRESULT_H
+
+
+#include <vector>
+
+namespace caf {
+/// Representation of CRUMBS MVA Slice ID output
+class SRCRUMBSResult {
+  public:
+  SRCRUMBSResult();
+  ~SRCRUMBSResult() {}
+
+  float score;
```

Comment:
Include units in comments (Ideally in doxygen comment strings with a brief description of each)